index.js

```diff
@@ -205,3 +205,30 @@ for (let j = 0; j < projects.length; j += 1) {
     });
   });
 }
+
+// form validation
+
+const form = document.getElementById('contact_me');
+const REQUIRED_MESSAGE = 'Inavlid email';
```

Comment:
**[OPTIONAL]** It would be better to guide the user on why the email is invalid to increase user experience means you can say `Email needs to be in lowercase` or something else which defines the error in more detail ✅